defaults/main.yml

```diff
@@ -30,6 +30,7 @@ go_waku_store_nodes: []
 go_waku_store_persist_messages: false
 go_waku_store_capacity: 10000
 go_waku_store_duration: '168h' # 7 days
+go_waku_store_vacuum: false
```

Comment:
I think I missed smth, why is off by default?

Reply:
Because we want to gradually enable it on one stage and then another, since it's a new feature.

Then we can make it true by default if we're sure that it works fine.